### PR TITLE
changed Button text Past to Best

### PR DESCRIPTION
**Pull request by Harsh97x**
* Replaced the button text from Past to Best in the menu.component.jsx line number 39
* PR closes the Issue#2

### CODE REVIEW

File: src/components/menu/menu.component.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ function Menu({ setIsNewStory }) {
         isActive={activeBtn === "past"}
         handleBtnClick={handleActiveBtn}
       >
-        Past
+        Best
```
Comment (Swastikyadav) [inline src/components/menu/menu.component.jsx:39]: This is fine!

File: package.json [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
-}
```
Comment (Swastikyadav) [inline package.json:38]: Why to do this, this issue does not require to make any change in package.json file.

File: src/components/menu/menu.component.jsx (hunk 1 above)
Comment (Swastikyadav) [reply src/components/menu/menu.component.jsx:39]: @Harsh97x raise a new PR only with the above change, and no changes in package.json file.

**Comment by Harsh97x** [comment]
Sorry for the novice mistakes, I'll correct them and raise a new PR.

**Comment by Swastikyadav** [comment]
Noworries, We all have been there.

**Pull request closed without merging**